tests/utils/formatDate.test.ts

```diff
@@ -1,7 +1,14 @@
 import MockDate from 'mockdate'
 import { toDate } from 'date-fns-tz'
+import { DateTime, Settings } from 'luxon'
```

Comment:
I'm getting a linting warning that `Settings` is imported but not used

Reply:
Yup! 😅 cleaned up in next PR!